### ISSUE TITLE
How to use with MC 1.18.2

### ISSUE THREAD

**Issue by JimmyBringus** [walterhiggins/ScriptCraft#463]
Is this project DOA? Or is it completed and works with all MC versions up to 1.18.2?

**Comment by Doomsdaybear** [comment]
No, the most recent code update was 3 years ago, so 1.18 Is out of the question. 1.16 and lower would work(go with lower to be safe).

**Comment by Doomsdaybear** [comment]
> Is this project DOA? Or is it completed and works with all MC versions up to 1.18.2?

But, if you want to try to make it work with 1.18, you could start a pull request

**Comment by ghost** [comment]
1.17 and onwards require the use of Java 16-17 which, don't have the JS engine this plugin uses (Nashorn.) You will have to find another Javascript in Minecraft plugin (they do exist)

**Comment by atomisadev** [comment]
This project does not support 1.18. Probably because it hasn't been updated for several years. I am thinking if I should work on maintaining this project myself.

**Comment by dogemanttv** [comment]
@atomisadev 
please do it
i dont want to use java :(
